build/utils.ts

```diff
@@ -313,11 +313,6 @@ export function makeTOC(doc) {
 }
 
 export function findPostFileBySlug(slug: string): string | null {
```

Comment:
This is still used in other places so we just remove the warning.
```suggestion
export function findPostFileBySlug(slug: string): string | null {
  if (!BLOG_ROOT) {
    return null;
  }
```